src/PerformanceTests/Tests/Categories/TransactionModesFixture.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using Tests.Permutations;
+using Variables;
+
+namespace Categories
+{
+    [TestFixture(Description = "Transaction Modes", Category = "Performance"), Explicit]
```

Comment:
Fixture must not be marked as `Explicit` or else it will not run on the build server.